packages/cli/package.json

```
@@ -26,9 +26,11 @@
 		"ora": "^8.2.0"
 	},
 	"devDependencies": {
+		"@types/fs-extra": "^11.0.4",
```

```suggestion
		"@types/fs-extra": "*",
```

and add the version in the top level `package.json`